examples/rest-xml-json/README_pt.adoc

```diff
@@ -0,0 +1,401 @@
+classe, então definir nos métodos, o método HTTP para usar (@GET, @POST, …).
+
+O Código
+
+== O serviço REST: @Path, @Produces, @Consumes
+
+Aqui nós vemos um bean que usa as anotações @Path, @Produces e @Consumes.  
+A anotação @Path identifica o modelo de caminho URI ao qual o recurso responde e é especificado no nível de classe de um recurso. O valor da anotação @Path 
+é um modelo de caminho de URI parcial relativo ao URI de base do servidor no qual o recurso está implementado, a raiz de contexto do WAR e o padrão de URL 
+para o qual o servlet auxiliar Jersey responde.
+A anotação @Produces é usada para especificar os tipos de mídia MIME ou representações que um recurso pode produzir e enviar de volta ao cliente. Se @Produces 
+for aplicado no nível da classe, todos os métodos em um recurso podem produzir os tipos MIME especificados por padrão. Se for aplicado no nível de método, substitui 
+quaisquer anotações @Produces aplicadas no nível da classe.
+A anotação @Consumes é usada para especificar quais tipos de mídia MIME de representações um recurso pode aceitar ou consumir do cliente. Se @Consumes for 
+aplicado no nível da classe, todos os métodos de resposta aceitarão os tipos MIME especificados por padrão. Se @Consumes for aplicado no nível do método, ele 
+substituirá qualquer @Consumes anotações aplicadas no nível da classe.
+
+Na realidade linhas:
+
+[código,java]
+----
+@Produces({ MediaType.APPLICATION_XML, MediaType.APPLICATION_JSON })
+@Consumes({ MediaType.APPLICATION_XML, MediaType.APPLICATION_JSON })
+----
+
+são opcionais, uma vez que é a configuração padrão. E essas linhas, podem
+ser configuradas pelo método também, se você precisar para ser mais preciso.
+
+[código,java]
+----
+@Path("/greeting")
+@Produces({ MediaType.APPLICATION_XML, MediaType.APPLICATION_JSON })
+@Consumes({ MediaType.APPLICATION_XML, MediaType.APPLICATION_JSON })
+public class GreetingService {
+    @GET
+    public Response message() {
+        return new Response("Hi REST!");
+    }
+
+    @POST
+    public Response lowerCase(final Request message) {
+        return new Response(message.getValue().toLowerCase());
+    }
+}
+----
+
+== Testando
+
+=== Teste para o serviço JAXRS
+
+O teste utiliza o OpenEJB ApplicationComposer para torná-lo trivial.
+
+A idea é primeiro ativar os serviços jaxrs. Isso é feito usando a 
+anotação @EnableServices.
+
+Em seguida, criamos rapidamente o aplicativo, simplesmente retornando um objeto
+representando o web.xml. Aqui nós, simplesmente usamos isso para definir o contexto
+root, mas você pode usá-lo para definir sua Aplicação REST também. E para 
+finalizar a definição da aplicação, nós adicionamos a anotação @Classes, para definir
+o grupo de classes para ser usado nessa aplicação.
+
+Finalmente para testá-lo, usamos a API cliente cxf, para chamar o serviço 
+REST em métodos get() e post().
+
+Notação lateral: para mostrar que usamos JSON ou XML, dependendo no método de teste nós
```

Comment:
```suggestion
SIDE NOTE: para mostrar que usamos JSON ou XML, dependendo no método de teste nós
```

Reply:
In portuguese do not use the word "Side Note", uses "Note" or "Observation", i made a mistake, in portuguese to say  "Notação" or "Observação", and not "Notação lateral" as i wrote. 
Do you like better, "Note"(Notação) or "Observation"(Observação) ?

Reply:
Sorry. I was thinking that "SIDE NOTE" was a tag. IMHO, "Observação" is better.

Reply:
The change is applied.